lib/formalist/child_forms/builder.rb

```diff
@@ -0,0 +1,45 @@
+require "json"
+require_relative "child_form"
+
+module Formalist
+  module ChildForms
+    class Builder
+      attr_reader :embedded_forms
+
+      def initialize(embedded_form_collection)
+        @embedded_forms = embedded_form_collection
+      end
+
+      def call(input)
+        return input if input.nil?
+
+        input = input.is_a?(String) ? JSON.parse(input) : input
```

Comment:
Seeing this line repeated across a bunch of the classes in this PR... I wonder if we can reasonably expect the `input` to _already_ be deserialized JSON before it gets passed here? I forget the details... maybe it's not possible because the JSON is deeply nested in some existing structure and therefore can't be deserialized sooner?

Reply:
This may be the case with rich text embeds, but shouldn't be in these many forms related classes, at least in the way we are using them. Thanks, will remove!